packages/react-styles/src/css/components/LogViewer/loggerToolbar.styles.css

```diff
@@ -0,0 +1,22 @@
+.logger__toolbar{
+  border: 1px solid var(--ins-c-logger--toolbar-BorderColor);
+  border-bottom: 0px;
+}
+
+.logger__toolbar--hasSearchbar {
+  height: 200px;
+}
+
+.logger__toolbar--icons {
+  padding: 6px 16px !important;
+}
+
+.toolbar__searchbar {
+    width: 500px;
+}
+
+.searchbar__btn {
+    width: 44px;
+    height: 36px;
+} 
```

Comment:
Name all your CSS selectors according to BEM standards. You can use [AccessConsoles](https://github.com/patternfly/patternfly-react/tree/master/packages/react-styles/src/css/components/Consoles) as an example. Everything should be scoped to a `.pf-c-logger` so as not to collide with consumer class names.